setup.sh

```diff
@@ -56,31 +54,12 @@ py3-pip-bash-completion
 py3-pip-doc
 "
 
-UTILITY_PROGRAMS="
-curl
-ffmpeg
-htop
-lynx
-mediainfo
-openssh
-p7zip
-rsync
-rtmpdump
-samba-common
-speedtest-cli
-"
-
-PIP_PROGRAMS="
-youtube-dl
-"
-
+UTILITY_PROGRAMS
 echo "START: Setup script"
 echo ""
 
-echo "START: Install Alpine Linux apk"
-grep -v "file:///ish/apk/" /etc/apk/repositories | dd of=/etc/apk/repositories bs=4194304
-echo https://dl-cdn.alpinelinux.org/alpine/v3.12/main >> /etc/apk/repositories
-echo https://dl-cdn.alpinelinux.org/alpine/v3.12/community >> /etc/apk/repositories
+echo https://dl-cdn.alpinelinux.org/alpine/v3.13/main >> /etc/apk/repositories
+echo https://dl-cdn.alpinelinux.org/alpine/v3.13/community >> /etc/apk/repositories
```

Comment:
```suggestion
echo "START: Install Alpine Linux apk"
echo https://dl-cdn.alpinelinux.org/alpine/v3.12/main >> /etc/apk/repositories
echo https://dl-cdn.alpinelinux.org/alpine/v3.12/community >> /etc/apk/repositories
sed -i -e '/http:\/\/apk.ish.app/d' /etc/apk/repositories 
```

The [iSH wiki](https://github.com/ish-app/ish/wiki/Using-Alpine-Linux-repositories) still points to 3.12 so this should remain in line with the iSH project itself. Once the project updates to a newer version then this script will be brought up in step with it